SETUP/tests/jsTests/formatPreviewTests.js

```diff
@@ -0,0 +1,113 @@
+/* global QUnit analyse */
+
+QUnit.test("Format preview test", function( assert ) {
+    let inLineTags = "[ibfg]|sc";
+    let suppress = {};
+    let text;
+    let issArray;
+
+    // assert start, length code and type correct for issue at index
+    function issueTest(index, start, length, code, type) {
+        let issue = issArray[index];
+        assert.deepEqual(issue.start, start);
+        assert.deepEqual(issue.len, length);
+        assert.deepEqual(issue.code, code);
+        assert.deepEqual(issue.type, type);
+    }
+
+    // missing inline start tag
+    text = "df</b> ab";
```

Comment:
I think it makes more sense to make each one of these their own QUnit.test case, no?

Reply:
Maybe, I was tring to make it more compact.